opendir_sample.c

```diff
@@ -17,7 +17,11 @@
  * 若读取目录成功，返回该目录结构，若读取目录失败，返回RT_NULL。
 */
 #include <rtthread.h>
-#include <dfs_posix.h> /* 当需要使用文件操作时，需要包含这个头文件 */
+#if RT_VER_NUM >= 0x40100
+#include <sys/_default_fcntl.h> /* 当需要使用文件操作时，需要包含这个头文件 */
```

Comment:
看看这个头文件可否编译过，这个是标准头文件
```suggestion
#include <fcntl.h> /* 当需要使用文件操作时，需要包含这个头文件 */
```

Reply:
感谢您提供的思路，这个标准头文件可以编译通过。
之前选用sys/_default_fcntl.h头文件的原因是，首先这两个头文件都可以通过编译，其次sys/_default_fcntl.h包含于fcntl.h中，是最小的头文件。
不过由于fcntl.h是标准头文件，更符合项目规范，而sys/_default_fcntl.h并不是，因此您提出的问题我认为是非常有价值的，我已经在最新的版本中修改完毕